libs/ts-rest/nest/src/lib/ts-rest-nest-handler.ts

```diff
@@ -336,14 +336,26 @@ export class TsRestHandlerInterceptor implements NestInterceptor {
       map(async (impl) => {
         let result = null;
         try {
-          const res = {
+          const implParams = {
             params: paramsResult.data,
             query: queryResult.success ? queryResult.data : req.query,
             body: bodyResult.success ? bodyResult.data : req.body,
             headers: headersResult.success ? headersResult.data : req.headers,
           };
 
-          result = routeKey ? await impl[routeKey](res) : await impl(res);
+          result = routeKey
+            ? await impl[routeKey](implParams)
+            : await impl(implParams);
+
+          /**
+           * If during the implementation the response was sent (e.g. a res.redirect or res.send was called)
```

Comment:
hmmmm, would this be intended behavior? Wouldn't this defeat the purpose of using a ts-rest handler anyways?